security/container_security/security-container-signature.adoc

```diff
@@ -0,0 +1,17 @@
+[id="security-container-signature"]
+= Container image signatures
+
+Red Hat delivers signatures for the images in the Red Hat Container Registries. Those signatures can be automatically verified when being pulled to {product-title} 4 clusters by using the Machine Config Operator (MCO).
+
+link:https://quay.io/[Quay.io] serves most of the images that comprise {product-title}, and only the release image is signed. Release images refer to the approved {product-title} images, offering a degree of protection against supply chain attacks. However, some extensions to {product-title} such as logging, monitoring, and service mesh are shipped as Operators from the Operator Lifecycle Manager (OLM). Those images ship from the link:https://catalog.redhat.com/software/containers/explore[Red Hat Ecosystem Catalog Container images] registry.
+
+To verify the integrity of those images between Red Hat registries and your infrastructure, enable signature verification.
+
+//Enabling Signature Verification in OCP
+include::modules/containers-signature-verify-enable.adoc[leveloffset=+1]
+
+//Verifying that the enable signature verification is active
+include::modules/containers-signature-verify-application.adoc[leveloffset=+1]
+
+.Additional resources
```

Comment:
"Additional resources" doesn't use the module style in assemblies. See https://github.com/openshift/openshift-docs/blob/master/contributing_to_docs/doc_guidelines.adoc#anchoring-prerequisites-additional-resources-and-next-steps-titles-in-assemblies and the section that immediately follows it.